source/patterns/@aws-solutions-constructs/aws-s3-step-function/README.md

```diff
@@ -22,7 +22,7 @@
 
 This AWS Solutions Construct implements an Amazon S3 bucket connected to an AWS Step Function.
 
-*Note - This construct uses Amazon EventBridge (Amazon CloudWatch Events) to trigger AWS Step Functions. EventBridge is more flexible, but triggering Step Functions with S3 Event Notifications has less latency and is more cost effective. If cost and/or latency is an issue, you should consider deploy aws-s3-lambda and aws-lambda-stepfunctions in place of this construct.*
+*Note - This construct uses Amazon EventBridge (S3 Event Notifications) to trigger AWS Step Functions State Machine executions. EventBridge is more flexible, but triggering executions with S3 Event Notifications has less latency and is more cost effective. If cost is an issue, you should consider deploying aws-s3-lambda and aws-lambda-stepfunctions in place of this construct.*
```

Comment:
This is confusing, because everything is now directly dependent upon S3 Event Notifications, and we use that term to discuss both solutions. Reading between the lines it sounds like the options are:

* Send S3 Event Notifications to EventBridge (what this now does) - more flexible, more expensive, more latency
* Trigger a Lambda function from S3 Event Notifications, have Lambda function invoke the Step Functions state machine. Less flexible, more work, more cost effective, lower latency.

Let's rewrite this accordingly (and can we confirm that option 2 is still cheaper and faster?)